### ISSUE TITLE
New 3.1.0 issue -- jsdoc failing:

### ISSUE THREAD

**Issue by justin808** [babel/babel-eslint#85]
```
assets/javascripts/actions/CommentActions.js
   5:2  error  JSDoc syntax error  valid-jsdoc
  19:2  error  JSDoc syntax error  valid-jsdoc
  29:2  error  JSDoc syntax error  valid-jsdoc
  39:2  error  JSDoc syntax error  valid-jsdoc
```

Line 5 is the first comment line.

```javascript
class CommentActions {
  /**
   * Fetch comments from server.
   *
   * @param {String} url
   * @param {Boolean} displaySpinner
   * @return undefined
   */
  fetchComments(url, displaySpinner) {
    this.dispatch(displaySpinner);
    CommentsManager.fetchComments(url)
      .then((comments) => this.actions.updateComments(comments),
            (errorMessage) => this.actions.updateCommentsError(errorMessage));
  }
```

**Comment by sebmck** [comment]
It's probably because your JSDoc comments **are** invalid. It's `returns`, not `return`.

**Comment by justin808** [comment]
@returns also fails.

You can try this out here.

https://github.com/justin808/react-webpack-rails-tutorial/pull/33

```
cd client
bin/lint
```

**Comment by sebmck** [comment]
Yep, it was as I suspected. The errors are correct, your JSDoc is invalid. The valid JSDoc return annotation is `@returns {type}` not `@return type`
